### ISSUE TITLE
RadarWidget drawing issue on gui resize

### ISSUE THREAD

**Issue by RubenJacobse** [ProfHoekstra/bluesky#164]
(Issue seems similar to: https://github.com/ProfHoekstra/bluesky/issues/154)

On resizing the BlueSky window the original radarwidget view is not deleted but appears on top of the new (correct) view. 

Issue seems to persist until either:
- The 'nodes' bar is clicked
- The BlueSky console is maximized
- Any of the buttons on the main BlueSky menu bar is clicked (Note: none of the console or resize buttons seem to have any effect) 
- Mouse click outside of BlueSky gui occurs

Initial window state:
![start](https://user-images.githubusercontent.com/32992187/43534214-1f145d4c-95b7-11e8-924e-eac2c5907f3c.png)

Maximizing the window:
![maximize_window](https://user-images.githubusercontent.com/32992187/43534239-22ba5456-95b7-11e8-9a95-ebf26aee9690.png)

Manual resize by dragging:
![manual_resize](https://user-images.githubusercontent.com/32992187/43534241-2441725a-95b7-11e8-8b30-d29f160c1f4d.png)

**Comment by jooste** [comment]
I cannot replicate this bug on my mac. It may be windows specific, or specific to your machine. Are you up-to-date with the latest commit?

**Comment by RubenJacobse** [comment]
(Yes, the issue was present in latest commit.)

Issue is fixed on my machine by #165, I hope this does not break anything elsewhere.

**Comment by jooste** [comment]
What happens when you interact with the simulation (starting a scenario) or the view (panning/zooming)?

**Comment by RubenJacobse** [comment]
Hi Joost,

- Opening a scenario via the 'File' button on the menu bar seems to emit an event that causes the widget to be redrawn correctly. 
- Opening / reopening scenarios via the buttons in the BlueSky does not seem to have this effect
- When panning/zooming after resizing the artefact of the previous screen size is frozen while the rest of the screen responds normally. (When resizing by dragging this can lead to only having thin regions at the bottom and right-hand side of the screen that respond normally)

I've also discovered that on my machine the issue seems related to the windows theme that is being used. When using a Win7 'Aero' theme the issue does not occur, but when using 'Win7 Basic' it does. I also tried to make a small video showing the issue, but it somehow does not appear when using the recording program (regardless of Win7 theme used)...

For now I'll be using the Win7 Aero theme :)
